README.md

```diff
@@ -1,15 +1,24 @@
 # The e-Manifest App for EPA
 
-This is a Ruby/Sinatra/Angular app, with a Jekyll static site in the `_static` directory. The Jekyll site will be generated in the `public/` directory, which Sinatra routes to automatically.
-
-Don't edit the `public/` folder directly! Jekyll will overwrite everything when building the site.
-
 [View the live application »](https://e-manifest.18f.gov)
 
 [![Build Status](https://travis-ci.org/18F/e-manifest.svg?branch=master)](https://travis-ci.org/18F/e-manifest)
 [![Code
 Climate](https://codeclimate.com/github/18F/e-manifest/badges/gpa.svg)](https://codeclimate.com/github/18F/e-manifest)
 
+## Application architecture
+
+This is a Rails application with a Postgres database. The app contains both API
+endpoints and web views. The API is a JSON REST API. The web app and the API
+use the same database.
+
+ElasticSearch is used to search for manifest records. We chose ElasticSearch
```

Comment:
good to know! googled it, so not sure how I ended up with wrong result. updated at 3b3a6b5